pystoi/stoi.py

```diff
@@ -65,7 +69,10 @@ def stoi(x, y, fs_sig, extended=False):
     if x_spec.shape[-1] < N:
         warnings.warn('Not enough STFT frames to compute intermediate '
                       'intelligibility measure after removing silent '
-                      'frames. Returning 1e-5. Please check you wav files',
+                      'frames. At least {} seconds are required, '
+                      'but only {} were found. '
+                      'Returning 1e-5. Please check you wav files'
+                      .format(((N - 1) * N_FRAME + NFFT) / FS, x.shape[0] / FS),
```

Comment:
This seems unrelated. It's after removing silent frames, so the minimum audio cannot be known a priori.

Reply:
It means "The size after silent frames removal", I though it was implied by the previous sentence in the comment.